### PR TITLE
did basic setup to get the project running on my local machine

### PR DESCRIPTION
**Pull request by annkaz**
- run npm commands
- seeded db locally
- added some changes to a script commands

@elanadev @emmanuelmota @j-scola let me know what you think

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,4 +25,5 @@ build/
 dist/
 
 data
+./public/bundle.js
```
Comment (j-scola) [inline .gitignore]: hey Anna, looks like the bundle was not git ignored as the file is added to this pr

I don't have the full context here but perhaps it should be `public/bundle.js` in git ignore

**Review by j-scola — COMMENTED** [review]
hey @annkaz I added a comment on the .gitignore change you made. it doesn't look like the bundle was removed from the repo here.

File: .gitignore (hunk 1 above)
Comment (annkaz) [reply .gitignore]: not sure what's the problem, but I tried all the options and just submitted again and it's still there. was my personal preference but I think should be fine

Comment (j-scola) [reply .gitignore]: cool, I'm going to approve. If you want to remove from the repo, its a but of a pain, but you need to remove from the file system and commit the change to this branch. Up to you!

**Review by j-scola — APPROVED** [review]
gj!

File: .gitignore (hunk 1 above)
Comment (annkaz) [reply .gitignore]: thank you! yes, I tried to removing  it, adding to the gitignore and create again but it  still persists

**Pull request merged**